explorer/assistant/utils.py

```diff
@@ -127,9 +126,8 @@ def fits_in_window(string: str) -> bool:
 
 def build_prompt(request_data, included_tables):
     user_prompt = ""
-
-    db_vendor = get_valid_connection(request_data.get("connection")).vendor
-    user_prompt += f"## Database Vendor / SQL Flavor is {db_vendor}\n\n"
+    djc = DatabaseConnection.objects.get(id=request_data.get("connection")).as_django_connection()
```

Comment:
Is this guaranteed to return an object? If not, it should either `get_or_create` or do something defensive with `try/except`